ENIGMAsystem/SHELL/Graphics_Systems/OpenGL3/std.cpp

```diff
@@ -29,6 +29,10 @@ namespace enigma {
   const unsigned int gl_major = 3, gl_minor = 3;
   const std::string shader_header = "#version 330\n";
 
+  GLenum shadertypes[5] = {
```

Comment:
It's probably unwise to use an ordered collection of enum constants that's disparate between two systems. People might index into it badly.